simulator/ssh/auth.go

```diff
@@ -0,0 +1,39 @@
+package ssh
+
+import (
+	"fmt"
+
+	"golang.org/x/crypto/ssh"
+)
+
+const privKey = `-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Please add some comment here and let people know that we know what we're doing by having private key in the repo.